components/input/input.css

```diff
@@ -0,0 +1,115 @@
+form {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: 2rem 2.4rem;
+  border-radius: 8px;
+  gap: 0.8rem;
+  background-color: var(--white);
+}
+
+form label {
+  font-size: 12px;
```

Comment:
You can use rem or em for font size. It will help in making the site responsive.